src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java

```diff
@@ -21,21 +22,6 @@
   public static final String BOTTOM_BOXAD = "BOTTOM_BOXAD";
   public static final String FEED_BOXAD = "FEED_BOXAD";
 
-  static {
-    SLOT_SELECTORS = new HashMap<>();
```

Comment:
I believe that if you restore this unfortunate map and fix the remaining 4 failing jobs, so they use selectors from this map all jobs will pass.